src/Printer/CollectorMetadataPrinter.php

```diff
@@ -35,11 +35,9 @@
 
 final readonly class CollectorMetadataPrinter
 {
-    private Standard $printerStandard;
-
-    public function __construct()
-    {
-        $this->printerStandard = new Standard();
+    public function __construct(
+        private Printer $printer
```

Comment:
PHPStan complains if we try to instantiate this directly, it needs to be passed through dependency injection

Reply:
Feel free to create `new Standard` printer directly, as we lightweight. We don't need any special printer here